assets/js/modules/analytics-4/components/common/PropertySelect.test.js

```diff
@@ -90,9 +90,9 @@ describe( 'PropertySelect', () => {
 			registry.dispatch( MODULES_ANALYTICS ).setAccountID( ACCOUNT_CREATE );
 			registry.dispatch( STORE_NAME ).finishResolution( 'getProperties', [ ACCOUNT_CREATE ] );
 		} );
-		// ACCOUNT_CREATE is an invalid accountID (but valid selection), so ensure the select IS currently disabled.
-		expect( selectWrapper ).toHaveClass( 'mdc-select--disabled' );
-		expect( selectedText ).toHaveAttribute( 'aria-disabled', 'true' );
+
+		// ACCOUNT_CREATE is an invalid (but valid selection), so ensure the select is not rendered
+		expect( container ).toBeEmptyDOMElement();
```

Comment:
Just to note @eugene-manuilov that the changed functionality of using ` isValidAccountID` and `isValidPropertyID` results in this test changing

Just checking that's ok with you! :+1: